Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -2,7 +2,7 @@ import {Grid, GridItem, Box, Flex, Spacer, Text, Center, useColorMode, VStack, I
 import HeadingCell from "../Shared/HeadingCell";
 import CONSTANTS from "../../data-constants.json"
 import getColor from "../Shared/colors";
-import images from "../Motor_Motion/Icons/Images"
+import images from "./Icons/Images"
```

Comment:
These changes make sense, but they seem unrelated to the Poetry change? Unless there's a connection I missed